### ISSUE TITLE
Wrong buttons colors with physical keyboard navigation

### ISSUE THREAD

**Issue by dimitriz09** [Decathlon/vitamin-compose#57]
**Context**

https://developer.android.com/develop/ui/views/touch-and-input/keyboard-input/navigation

TL;DR

"Android supports physical keyboards attached to the device. A keyboard offers not only a convenient mode for text input, but also offers a way for users to navigate and interact with your app. [...], tablets and similar devices are growing in popularity and many users like to attach keyboard accessories."

**Describe the bug**
The selected state of a button makes the text unreadable.

**To Reproduce**
Connect a physical keyboard to your device (usb C, or bluetooth). 
Launch vitamin compose demo with your bare hands 
Navigate to button demo
Now use the keyboard, press "tab" to go from one button to the other

**Expected behavior**
When a button is selected (or has the "keyboard focus") its text should be readable

**Screenshots**
![Screenshot from 2022-12-14 09-28-25](https://user-images.githubusercontent.com/109512997/207544716-f6dababc-a249-44d3-ac21-62987b3345dc.png)
^ primary
![Screenshot from 2022-12-14 09-29-04](https://user-images.githubusercontent.com/109512997/207544872-d554fe5b-bcba-4c29-9723-7b55ed0b7c5c.png)
^ Ghost living up to his name 

**Smartphone (please complete the following information):**
All

**Comment by GerardPaligot** [comment]
For now, Vitamin Compose give a ripple implementation for the pressed state but we need to work on the state focus (keyboards) and hover (stylus or mouse on ChromeOS).

It'll be worked on design side and after that, we'll integrate it in Vitamin Compose and Android (poke @ManonPolle) implementations.

**Comment by FDELAHA24** [comment]
@Decathlon/design-system-core-team-design

**Comment by lauthieb** [comment]
Decision after grooming session: 
- @Decathlon/design-system-core-team-design will align & apply native standards for ripple (Figma Android UI Kit & doc) for buttons. 
- @Decathlon/design-system-core-contrib-android-compose-dev will use native standards for ripple (on Android & Compose implementations) for buttons.
